lesson3/Lesson3HW/app/src/main/res/layout/activity_main.xml

```diff
@@ -35,8 +38,6 @@
 
     </FrameLayout>
 
-    <include layout="@layout/content_main" />
-
```

Comment:
This was your problem initially, you had your main fragment statically in `content_main` and when you switched to the settings fragment in your FrameLayout it showed up underneath this `content_main`